vpp.env

```diff
@@ -9,7 +9,7 @@ VPP_BINAPI_1901=plugins/vpp/binapi/vpp1901
 VPP_IMG_1904=ligato/vpp-base:19.04
 VPP_BINAPI_1904=plugins/vpp/binapi/vpp1904
 # VPP 19.08-rc0
-VPP_IMG_1908=ligato/vpp-base:19.08-rc0.212-gf6c7aec95
+VPP_IMG_1908=ligato/vpp-base:v19.08-rc0~324-g2ecf18a55
```

Comment:
This is not correct. Please try `make dev-image` to verify. Also you can run `make verify-binapi VPP_VERSION=1908` to verify binapi is generated correctly.

Reply:
Thanks, I'd get to that later.